### DISCUSSION TITLE
Which country was called the "🦪 of 🌍"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#882] [Galaxy Trivia] [answered]
**Bonus:** Where does the name come from?

**Comment by peterbaccaro** [comment]
The country that has been referred to as the "Pearl of the World" is the **Philippines**. The phrase "Pearl of the World" was **used by early Spanish explorers and colonizers to describe the archipelago due to its natural beauty, abundant resources, and strategic location in Southeast Asia**. The Philippines is known for its stunning beaches, diverse flora and fauna, rich cultural heritage, and picturesque landscapes. It is home to numerous natural wonders, including beautiful islands, coral reefs, volcanoes, and lush forests.

**Reply by ParanoidUser** [reply]
If we look closely at the emoji 🌍, we can find another "pearl" in it. 🫢

**Reply by peterbaccaro** [reply, marked as answer]
The country referred to as the "Pearl of Africa" is **Uganda**. 

**The nickname was famously coined by Sir Winston Churchill, the British Prime Minister, during his visit to the country in 1907. He was captivated by the natural beauty and diverse landscapes that Uganda has to offer.**

**Reply by peterbaccaro** [reply]
Is that correct?

**Reply by ParanoidUser** [reply]
Bull! 🎯

**Comment by tshenolo** [comment]
Belgium

**Reply by ParanoidUser** [reply]
Miss... 🫢 Any other ideas?
